docs/tutorials/hello-world-application/using-the-database.mdx

```diff
@@ -227,9 +236,7 @@ Once this action has been executed, the database will contain the following data
 
 ### select_greetings
```

Comment:
@brennanjl, do you think this should stay here or move to the same section where I cover the `.selectQuery()` endpoint (below)?

Reason to keep it here would be that this section is `Action Execution`. Reason to move it would be that all the other actions in this section use the broadcast endpoint, but the `select_greetings` action is a `view` action and uses the `call` endpoint. LMK what you think.

Reply:
I think here is fine.  Makes more sense.  From the perspective of the client, we are calling an action, which belongs here and not with ad-hoc select